.github/workflows/examples.yml

```
@@ -16,79 +16,79 @@
+      - id: get-sha
+        run: echo "::set-output name=sha::$(git rev-parse HEAD)"
+      - name: Install Node modules with Yarn for renderer package
+        run: |
+          yarn install --no-progress --no-emoji
+          sudo yarn global add yalc
+      - name: yalc publish for react-on-rails
+        run: yalc publish
+      - name: Install Ruby Gems for package
+        run: bundle lock --add-platform 'x86_64-linux' && bundle check --path=vendor/bundle || bundle  _2.5.9_ install --path=vendor/bundle --jobs=4 --retry=3
+      - name: Ensure minimum required Chrome version
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2071:error:4:39: < is for string comparisons. Use -lt instead

(shellcheck)